hypershift-operator/controllers/scheduler/dedicated_request_serving_nodes.go

```diff
@@ -771,6 +771,12 @@ func (r *DedicatedServingComponentSchedulerAndSizer) updateHostedCluster(ctx con
 			delete(hc.Annotations, hyperv1.MachineHealthCheckTimeoutAnnotation)
 		}
 	}
+	if sizeConfig.Effects != nil && sizeConfig.Effects.MaximumRequestsInflight != nil {
+		hc.Annotations[hyperv1.KubeAPIServerMaximumRequestsInFlight] = fmt.Sprint(*sizeConfig.Effects.MaximumRequestsInflight)
```

Comment:
It's there any kind of validation around the annotation content?

Reply:
The CR ensures that it is a number. As far as limiting the values for that number, I'm not sure that we know just yet what those limits should be. This is an API that is meant to be consumed by the service provider (ie. fleet manager), so I'm less worried about validation here.